### ISSUE TITLE
ckan is not reachable after reinstallation #19

### ISSUE THREAD

**Issue by Jan-Marcek** [OpenDataNode/ckanext-odn-cas#2]
steps to reproduce
1. login into ic (don't close browesr)
2. reinstall odn
3. (on the same browser) go to page: host/internalcatalog - server returns 500 and a user is unable to work with ic

stack trace from `/var/log/apache2/odn-ckan-ic.error.log`

```
May 06 11:26:46 2015] [error] [client 127.0.0.1]   File "/usr/share/python/odn-ckan-shared/lib/python2.7/site-packages/beaker/middleware.py", line 73, in __call__
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]     return self.app(environ, start_response)
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]   File "/usr/share/python/odn-ckan-shared/lib/python2.7/site-packages/beaker/middleware.py", line 155, in __call__
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]     return self.wrap_app(environ, session_start_response)
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]   File "/usr/share/python/odn-ckan-shared/lib/python2.7/site-packages/routes/middleware.py", line 131, in __call__
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]     response = self.app(environ, start_response)
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]   File "/usr/share/python/odn-ckan-shared/lib/python2.7/site-packages/pylons/wsgiapp.py", line 125, in __call__
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]     response = self.dispatch(controller, environ, start_response)
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]   File "/usr/share/python/odn-ckan-shared/lib/python2.7/site-packages/pylons/wsgiapp.py", line 324, in dispatch
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]     return controller(environ, start_response)
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]   File "/usr/share/python/odn-ckan-shared/lib/python2.7/site-packages/ckan/lib/base.py", line 338, in __call__
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]     res = WSGIController.__call__(self, environ, start_response)
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]   File "/usr/share/python/odn-ckan-shared/lib/python2.7/site-packages/pylons/controllers/core.py", line 217, in __call__
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]     response = self._inspect_call(self.__before__)
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]   File "/usr/share/python/odn-ckan-shared/lib/python2.7/site-packages/pylons/controllers/core.py", line 107, in _inspect_call
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]     result = self._perform_call(func, args)
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]   File "/usr/share/python/odn-ckan-shared/lib/python2.7/site-packages/pylons/controllers/core.py", line 60, in _perform_call
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]     return func(**args)
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]   File "/usr/share/python/odn-ckan-shared/lib/python2.7/site-packages/ckan/lib/base.py", line 242, in __before__
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]     self._identify_user()
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]   File "/usr/share/python/odn-ckan-shared/lib/python2.7/site-packages/ckan/lib/base.py", line 271, in _identify_user
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]     item.identify()
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]   File "/usr/share/python/odn-ckan-shared/lib/python2.7/site-packages/ckanext/cas/plugin.py", line 142, in identify
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1]     name_first = self._get_first_value(user_data[self.roles_config.attr_name_first])
[Wed May 06 11:26:46 2015] [error] [client 127.0.0.1] KeyError: 'givenName'
```

**Comment by skrchnavy** [comment]
OpenDataNode/odn-box#19 closed in v1.0.2

**Comment by Jan-Marcek** [comment]
this is not fixed.

**Comment by mvi-eea-sk** [comment]
@Jan-Marcek: that error message means that the role.properties was not set properly
It couldn't find givenName attribute in the response from CAS

**Comment by Jan-Marcek** [comment]
@mvi-eea-sk  please try it. The problem is that the old cookies remain in browser.

**Comment by skrchnavy** [comment]
@peterklimo is this fixed in v1.1.1?

**Comment by Jan-Marcek** [comment]
@skrchnavy No, it's not.

**Comment by Jan-Marcek** [comment]
it seems to be fixed in 1.1.2
